fixtures/gists-app/scripts/install_remix.js

```diff
@@ -9,6 +9,9 @@ let installDir = path.resolve(__dirname, "../node_modules");
 async function run() {
   // Install all remix packages
   await fs.ensureDir(installDir);
+  await fs.emptyDir(path.resolve(installDir, "@remix-run"));
+  await fs.emptyDir(path.resolve(installDir, "remix"));
+  await fs.emptyDir(path.resolve(installDir, "create-remix"));
```

Comment:
```suggestion
  await fs.emptyDir(installDir);
```

All we need is to change this line from `fs.ensureDir(installDir)` to `fs.emptyDir(installDir)`. The `emptyDir` call with [both ensure the dir exists and that it's empty](https://github.com/jprichardson/node-fs-extra/blob/master/docs/emptyDir.md#emptydirdir-callback).